site.yml

```diff
@@ -128,6 +131,8 @@
       when: gce_account_certificate_pem is defined
     - name: copy bash_gpg
       copy: src=files/.bash_gpg dest=/var/lib/jenkins/.bash_gpg
+    - name: create jenkins gnupg hidden directory
+      file: path=/var/lib/jenkins/.gnupg state=directory owner=jenkins group=jenkins mode=0700
```

Comment:
The gpg setup was "originally" done as a manual step which we did by hand on the CI but we changed the implementation.